Dockerfile

```diff
@@ -1,4 +1,9 @@
-FROM nginx
+FROM gcr.io/triggermesh/mkdocs:latest AS builder
```

Comment:
Where is the source of that image, and what's the rational for not using the `python` image?

Reply:
note sure where the source lives, but you're right we can just replace it with plain old python image.

Reply:
updated!